src/code/qualification/ConvertTimeUnit.m

```diff
@@ -0,0 +1,22 @@
+function [Yout, YoutUnit] = ConvertTimeUnit(Yin, YinUnit, YoutUnit)

```

Comment:
Should be named `convertTimeUnit` no?

@Yuri05 @KatrinCoboeken I am sure we have something in place for unit conversion. Please advise